### PR TITLE
8266460: java.io tests fail on null stream with upgraded jtreg/TestNG

### PR DESCRIPTION
**Pull request by LanceAndersen**
Hi all,

This fix addresses a change in TestNG behavior for the Before/AfterGroups annotations that was introduced  via  https://github.com/cbeust/testng/pull/2167.   The tests have been verified against the latest TestNG release and continue to run with the current TestNG release used by jtreg

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8266460](https://bugs.openjdk.java.net/browse/JDK-8266460): java.io tests fail on null stream with upgraded jtreg/TestNG

### Reviewers
 * [Brian Burkhalter](https://openjdk.java.net/census#bpb) (@bplb - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/3866/head:pull/3866` \
`$ git checkout pull/3866`

Update a local copy of the PR: \
`$ git checkout pull/3866` \
`$ git pull https://git.openjdk.java.net/jdk pull/3866/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 3866`

View PR using the GUI difftool: \
`$ git pr show -t 3866`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/3866.diff">https://git.openjdk.java.net/jdk/pull/3866.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/io/InputStream/NullInputStream.java [hunk 1]
```diff
@@ -21,13 +21,15 @@
...
 import java.io.ByteArrayOutputStream;
 import java.io.EOFException;
-import java.io.InputStream;
 import java.io.IOException;
-import org.testng.annotations.AfterGroups;
-import org.testng.annotations.BeforeGroups;
-import org.testng.annotations.Test;
+import java.io.InputStream;
+
 import static org.testng.Assert.*;
```
Comment (bplb) [inline test/jdk/java/io/InputStream/NullInputStream.java:33]: Would it not be more standard to put the new imports just after this import? Same comment applies in the other files.

Comment (LanceAndersen) [reply test/jdk/java/io/InputStream/NullInputStream.java:33]: That's IntelliJ magic.  I can update if you prefer and can let IntelliJ optimize all of the imports

Comment (bplb) [reply test/jdk/java/io/InputStream/NullInputStream.java:33]: Ah. I was just thinking of consistency with other tests. Up to you.

Comment (LanceAndersen) [reply test/jdk/java/io/InputStream/NullInputStream.java:33]: I think your milage will vary depending on the author and the IDE being used.  My primary concern was to address the issue for the failing test and Intellij arranged the imports as they are above.  I guess I am less concerned about the imports as I think in some cases it comes to personal preference.

So I would prefer to leave as is if you are OK :-)

Comment (bplb) [reply test/jdk/java/io/InputStream/NullInputStream.java:33]: I think that's fine.

**Review by bplb — APPROVED** [review]

**Pull request closed without merging**